packages/react-core/src/components/Pagination/examples/PaginationSticky.tsx

```diff
@@ -5,6 +5,7 @@ export const PaginationSticky: React.FunctionComponent = () => {
   const [page, setPage] = React.useState(1);
   const [perPage, setPerPage] = React.useState(20);
```

Comment:
Wondering if we should set the `perPage` value to 100 here since we probably want this demo to be scrollable by default... That way the sticky pagination functionality will be evident in the demo!
```suggestion
  const [perPage, setPerPage] = React.useState(100);
```